### DISCUSSION TITLE
Portainer option with BorgBackup

### DISCUSSION THREAD

**Discussion by foxhoundv** [nextcloud/all-in-one#3076] [Questions] [answered]
Is there a way to add BorgBackup as a container?

I copied the portainer stack information but I thought it would include BorgBackup.

![image](https://github.com/nextcloud/all-in-one/assets/16512250/1d8712d0-64cb-444f-9258-f6db6c233c19)

**Comment by szaimen** [comment]
Hi, I dont understand your question. Can you please paraphrase it? Thanks!

**Comment by foxhoundv** [comment]
In one server I have, I ran Nextcloud-AIO using "docker run...", in another server, I was using Portainer with the compose file.

With "docker run..." the following containers start:
- nextcloud-aio-watchtower
- nextcloud-aio-talk
- nextcloud-aio-redis
- nextcloud-aio-onlyoffice
- nextcloud-aio-notify-push
- nextcloud-aio-nextcloud
- nextcloud-aio-mastercontainer
- nextcloud-aio-imaginary
- nextcloud-aio-database
- nextcloud-aio-borgbackup*
- nextcloud-aio-apache

With the Portainer Compose file, the following containers start:
- nextcloud-aio-talk
- nextcloud-aio-redis
- nextcloud-aio-onlyoffice
- nextcloud-aio-notify-push
- nextcloud-aio-nextcloud
- nextcloud-aio-mastercontainer
- nextcloud-aio-database
- nextcloud-aio-apache

I didn't enable imaginary so that is understandable, however there is no borgbackup container, and there doesn't seem to be an option to enable it.

**Comment by foxhoundv** [comment]
![image](https://github.com/nextcloud/all-in-one/assets/16512250/7e32056e-912a-4337-8db8-7702b25a1f77)

In the backup section of the AIO interface, this is what it says...
![image](https://github.com/nextcloud/all-in-one/assets/16512250/3b494e80-49a7-428e-864a-e9963fe3c7bc)

Do I have to submit a backup location before it will create a borgbackup container?

**Comment by szaimen** [comment, marked as answer]
> Do I have to submit a backup location before it will create a borgbackup container?

Yes :)

**Reply by foxhoundv** [reply]
Awesome, thank you. I thought the BorbBackup container was created by default. I just realized the logic in doing it this way!
